usage/bigquery.go

```diff
@@ -99,6 +99,13 @@ func buildUsageSummaryQuery(table string, userID string, creatorID string, spec
 		query = query.Where("creator_id = ?", creatorID)
 	}
 
+	if from := spec.From; from != nil {
+		query = query.Where("time >= timestamp_millis(?)", from.UnixMilli())
+	}
+	if to := spec.To; to != nil {
+		query = query.Where("time < timestamp_millis(?)", to.UnixMilli())
+	}
```

Comment:
**nit:** BigQuery lib supports `time.Time` natively
```suggestion
	if from := spec.From; from != nil {
		query = query.Where("time >= ?", from)
	}
	if to := spec.To; to != nil {
		query = query.Where("time < ?", to)
	}
```

Reply:
Oh nice! I see that it's like this also on the viewership bigquery, may change that as well at this point